src/utils/wallet.js

```diff
@@ -41,13 +46,32 @@ export class Wallet {
     this.account = account;
   }
 
-  static getAccountFromSeed(seed, walletIndex, accountIndex = 0) {
-    const derivedSeed = bip32
-      .fromSeed(seed)
-      .derivePath(`m/501'/${walletIndex}'/0/${accountIndex}`).privateKey;
+  static getAccountFromSeed(
+    seed,
+    walletIndex,
+    derivationPath = undefined,
+    accountIndex = 0,
+  ) {
+    const path = Wallet.derivationPath(
+      walletIndex,
+      derivationPath,
+      accountIndex,
+    );
+    const derivedSeed = bip32.fromSeed(seed).derivePath(path).privateKey;
     return new Account(nacl.sign.keyPair.fromSeed(derivedSeed).secretKey);
   }
 
+  static derivationPath(walletIndex, derivationPath, accountIndex) {
+    switch (derivationPath) {
+      case DERIVATION_PATH.deprecated:
+        return `m/501'/${walletIndex}'/0/${accountIndex}`;
+      case DERIVATION_PATH.bip44:
+        return `m/44'/501'/${walletIndex}'/${accountIndex}`;
```

Comment:
This doesn't use the "change" parameter, as suggested by https://github.com/project-serum/spl-token-wallet/issues/9, although I don't fully understand why this is desireable. @garywang thoughts?

Alternatively, we could change this to `m/44'/501'/${walletIndex}'/0/${accountIndex}`. See https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels.

Reply:
Let's get rid of the `accountIndex` parameter, since we no longer use it and it's always set to 0.

Reply:
So just `m/501'/${walletIndex}'/0/0` and `m/44'/501'/${walletIndex}'/0`

Reply:
Sounds good. 

Also related, we probably need to fork the bip32 package (or just a different one) due to this https://github.com/project-serum/spl-token-wallet/issues/9#issuecomment-739135179.

Reply:
bip32 specifies "Bitcoin seed" as the hmac key for the master key https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#master-key-generation

Reply:
You're right. But I think if we want to be consistent, with ledger + trezor/crypto, we need to use `ed25519 seed`, instead of `Bitcoin seed` https://github.com/trezor/trezor-crypto/blob/915b3dbbbf58c262865647728a3463b8785fc965/bip32.c#L56.

I still need to check these changes against other wallets to validate this.

Reply:
BIP32 assumes secp256k1, whereas we want ed25519. 

According to [SLIP-10](https://github.com/satoshilabs/slips/blob/master/slip-0010.md#body"), "for other curves it uses a different salt than BIP-0032. This avoids using the same private key for different elliptic curves with different orders." 

So we need to use "ed25519 seed" as the hmac key, according to that document, which is consistent with the comments in #9.

Reply:
Right, when we switch to ed25519 we should also switch the hmac key.